### PR TITLE
fix(Bumper Page): Prevent clicks from registering

### PR DESCRIPTION
**Pull request by gunhansancar**
## JIRA
[AAG-3038]

## DESCRIPTION
fix(Bumper Page): Prevent clicks from registering when the user clicks outside the bumper page

## SCREENSHOTS
n/a

[AAG-3038]: https://superawesomeltd.atlassian.net/browse/AAG-3038?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -67,6 +67,7 @@ dependencies {
     androidTestImplementation "androidx.test:core-ktx:1.4.0"
     androidTestImplementation "androidx.test.espresso:espresso-core:$espresso_core_version"
     androidTestImplementation "androidx.test.espresso:espresso-intents:$espresso_core_version"
+    androidTestImplementation 'androidx.test.uiautomator:uiautomator:2.2.0'
```
Comment (me2k) [inline app/build.gradle:70]: Interesting that you've added this, I added it in my UI Test CI branch but I didn't think it did anything.

File: app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt [hunk 2]
```diff
@@ -20,13 +22,11 @@ import tv.superawesome.demoapp.interaction.CommonInteraction
 import tv.superawesome.demoapp.interaction.ParentalGateInteraction
 import tv.superawesome.demoapp.interaction.SettingsInteraction
 import tv.superawesome.demoapp.model.TestData
+import tv.superawesome.demoapp.util.*
```
Comment (me2k) [inline app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt:25]: Do you like wild card imports? If I were to turn on Detekt it has a default rule that rejects these...

**Review by me2k — APPROVED** [review]
Great job 👏  We really need to get rid of the base module. There are a lot of changes here 😬

File: app/build.gradle (hunk 1 above)
Comment (gunhansancar) [reply app/build.gradle:70]: It is required for `UiDevice.getInstance(getInstrumentation()).click(0, 100)` which allows to directly interact with the device and click anywhere on the screen.

File: app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt (hunk 2 above)
Comment (gunhansancar) [reply app/src/androidTest/java/tv/superawesome/demoapp/BannerUITest.kt:25]: I use auto import so this has to be done automatically I guess. I can update if you prefer

**Pull request merged**